src/Service/ScopeHintService.php

```diff
@@ -104,6 +115,9 @@ public function getConfigValuesForScopes(
             }
         }
 
+        $configValues[0]["values"] = str_replace('\n', ' ', $configValues[0]["values"]);
```

Comment:
Usually, we only use single quotation marks in PHP code. An exception would be `"\n"`, which doesn't work if it is `'\n'`. I am interested: does it work here?
So, I'd use single quotation marks where you use double quotation marks and vice versa in the code block.